mysite/announcements/views.py

```diff
@@ -0,0 +1,45 @@
+from django.shortcuts import render
+from django.contrib.auth.models import User, Group
+from django.contrib.auth import authenticate, login
+from django.http import HttpResponseRedirect
+from django import forms
+from .forms import UserRegistrationForm
+from .models import Announcements
+
+# Create your views here.
+
+
+def home(request):
+    user_id = request.user.id
+
+    current_user_groups = User.objects.filter(id=user_id)[0].groups.all()
+    valid_announcements = Announcements.objects.filter(groups__in=current_user_groups)\
+        .exclude(sent_at=None).filter(has_expired=False)
+
+    context = {
+        'user_id': user_id,
+        'valid_announcements': valid_announcements
+               }
+    return render(request, 'announcements/home.html', context)
+
+
+def register(request):
+    if request.method == 'POST':
+        form = UserRegistrationForm(request.POST)
+        if form.is_valid():
+            userObj = form.cleaned_data
```

Comment:
We don't use camel case for naming variables in python. Please read https://www.python.org/dev/peps/pep-0008/#naming-conventions

Reply:
My bad, I copied a little code segment from a resource and didn't notice. I'll be more aware next time